zanata-war/src/test/java/org/zanata/service/impl/TranslationUpdatedManagerTest.java

```diff
@@ -100,7 +100,7 @@
     private LocaleId localeId = LocaleId.DE;
     private String versionSlug = "versionSlug";
     private String projectSlug = "projectSlug";
-    private int wordCount = 10;
+    private Long wordCount = 10L;
```

Comment:
If you have to call intValue() on it all the time, perhaps wordCount should stay an int (or a long).